### DISCUSSION TITLE
Drive Categories

### DISCUSSION THREAD

**Discussion by Dulanic** [teslamate-org/teslamate#505] [Ideas]
I'd love to see the ability to add categories to drives. This would be super helpful tracking different types of trips you take between personal, business etc. and I am sure there would be other use cases.

**Describe the solution you'd like**

I'd love to see the ability to categorize drives. I am unsure if grafana has write abilities, but if not should be doable /w js/html.

**Describe alternatives you've considered**

This option is available within TeslaFi, but I prefer to keep data private.

**Comment by Zebble** [comment]
+1 for this feature as well.  Some sort of tagging would be really valuable, possibly something that could also auto-tag based on destination or combo of destination+origination?

**Comment by roadrash2108** [comment]
+1 because this would give the foundation to distingish between private and business trips

**Comment by SigLinJo** [comment]
It could maybe be added as an automatic tagging function coupled to a property set for geo-fences? Could be a neat feature to have! combined with a feature to create a trip with drives filtered for a specific category and the report for driving cost reimbursement or similar use cases would be as good as ready to submit :)

**Reply by Cannonballitk** [reply]
+1 this is the only reason why i have Teslafi

**Comment by MartinSaaby** [comment]
Any news on adding a tag to a trip "native" in Teslamate?

**Reply by Knuppel1983** [reply]
> Any news on adding a tag to a trip "native" in Teslamate?

Same request. We need this.

**Reply by brymonster** [reply]
Would love to have drive categorization!
